### ISSUE TITLE
go 1.4 dependency

### ISSUE THREAD

**Issue by erkules** [docker/docker-network#2]
```
$ go version
go version go1.3.2 linux/amd64

$ go get github.com/docker/docker-network
# github.com/docker/docker-network/Godeps/_workspace/src/golang.org/x/sys/unix
../GOPATH/src/github.com/docker/docker-network/Godeps/_workspace/src/golang.org/x/sys/unix/asm.s:6 6a: No such file or directory: textflag.h
```

Works with go 1.4. Maybe worth mentioning.

**Comment by erikh** [comment]
Yeah I'm working on this now. Closing this.

**Comment by LK4D4** [comment]
Two question: why close not resolved issue? And why support 1.3.2?

**Comment by erikh** [comment]
Good question, but I felt it would be solved by my patch, no?
